templates/scenarios/command-and-response-ts/bot/README.md

```diff
@@ -153,7 +153,7 @@ import { HelloWorldCommandHandler } from "../helloworldCommandHandler";
 import { DoSomethingCommandHandler } from "../doSomethingCommandHandler";
 import { ConversationBot } from "@microsoft/teamsfx";
 
-const commandBot = new ConversationBot({
```

Comment:
In notification samples, we use the following code:
```
export const bot = new ConversationBot({
...
});
```
Shall we unify the naming for all `ConversationBot` in our samples?

Reply:
Thanks, will change the variable name to notificationApp in notification templates. There's also feedback about renaming ConversationBot, we can track the SDK changes separately.